src/navier_stokes/INSVCStaggeredHierarchyIntegrator.cpp

```diff
@@ -438,14 +438,30 @@ INSVCStaggeredHierarchyIntegrator::INSVCStaggeredHierarchyIntegrator(const std::
                                  << "  valid choices are: VC_HARMONIC_INTERP, VC_AVERAGE_INTERP\n");
     }
 
-    // Get the scaling coefficient
-    d_A_scale = 1.0;
-    if (input_db->keyExists("condition_no_scaling")) d_A_scale = input_db->getDouble("condition_no_scaling");
-    if (d_A_scale <= 0.0)
+    // Get the scaling coefficients
+    if (input_db->keyExists("condition_no_scaling"))
```

Comment:
Does this actually make a difference in practice?

Reply:
Yes, in extreme cases of viscosity and density ratios

Reply:
Interesting.

Can we change the key name to something like `operator_scale_factors`? I think that `condition_no_scaling` is a little bit opaque. If you want to use condition number in the name, please switch it to `num` instead of `no` --- `condition_no_scaling` makes it seem like "no scaling" is being done, when it is in fact the opposite.

Reply:
Makes sense. `operator_scale_factors` makes it more clear. @nishantnangia : make this change when you get a chance. I could miss a spot or two.

Reply:
Will do.